### ISSUE TITLE
Clarify usage of quotes in declarative bindings

### ISSUE THREAD

**Issue by MaKleSoft** [Polymer/lit-html#507]
Its looks like
```html
<my-element .prop=${prop}></my-element>
```
and
```html
<my-element .prop="${prop}"></my-element>
```
produce the same results but it's unclear if and how the use of quotes changes the behavior of the binding. If this is documented somewhere, I haven't seen it. I've seen both versions used in documentation [0][1] without any explanation of what the differences are or which syntax is recommended:

[0] https://github.com/Polymer/lit-element#a-simple-base-class-for-creating-custom-elements-rendered-with-lit-html
[1] https://polymer.github.io/lit-html/guide/writing-templates.html#binding-types

**Comment by mkay581** [comment]
Yeah, [I brought this same thing up in the lit-html slack channel the other day](https://polymer.slack.com/archives/C6TEC35M4/p1536587711000100). The response I got doesn't really explain the differences, but it at least helped me pick the recommended way going forward.

**Comment by aadamsx** [comment]
Can I get an invite to this channel?

**Comment by MaKleSoft** [comment]
For reference (since not everyone has access to the slack channel):

> **mark** [3:55 PM]
should we be using quotes when declaring bindings? Or are they optional? If optional, is this temporarily the case? Or will one of them no longer be supported in the future? I know its hard to see into the future, but I'd like to avoid having to refactor something like this later on, which is generally hard to regex a search and replace for :sweat_smile:

> **43081j** [4:02 PM]
@mark you don't need them and afaik its recommended you don't have them

Would be nice to get an official answer though. Also, I'm not sure how I feel about leaving syntax choices up to individual preference. Look where that got us with javascript...

**Comment by justinfagnani** [comment]
The "official" answer is that quotes only matter if you have an interpolation of expressions and string literals. Like HTML, the quotes are optional in lit-html in some situations. This is because we just use the browser's HTML parser.

**Comment by abraham** [comment]
@aadamsx you can Join the Polymer slack here: https://polymer-slack.herokuapp.com/

**Comment by akc42** [comment]
Currently the vscode lit-html syntax highlighter doesn't highlight unless the quotes are there.  So I am using them regardless.  I hope this is the right thing?
